src/components/new-request/search.vue

```diff
@@ -212,7 +182,43 @@
       </v-col>
     </v-row>
 
-    <div v-if="!isFederal" class="mt-3 text-center">
+    <!-- Bullets Colin Link component. Show only if a business type is selected. -->
+    <BulletsColinLink
+      v-if="entity_type_cd"
+      :businessType="entity_type_cd"
+      :colinButton="showColinButton"
+      :showDesignation="showDesignationSelect"
+      @radioButtonChange="selectedCompanyType = $event"
+    >
+      <template v-slot:name-input-slot>
+        <NameInput
+          v-if="!isFederal"
+          :is-mras-search="(getIsXproMras && !noCorpNum)"
+          :menu-props="{ bottom: true, offsetY: true}"
+          id="name-input-component"
+          class="pt-3"
+          @emit-corp-num-validity="corpNumValid = $event"/>
+        <p v-else class="pl-3 text-body-2">Federally incorporated businesses do not need a Name Request. You may
+          register your extraprovincial business immediately using its existing name at Corporate Online.</p>
+      </template>
+      <template v-slot:designation>
+        <v-col md="4" lg="4">
```

Comment:
You probably don't need the `lg` -- how this works is columns are 12 wide (unless you declare them differently) and are 4 wide from `md` size and up.